windows/src/buildtools/mkver/Main.pas

```diff
@@ -332,6 +332,14 @@ procedure UpdateFile(const fin, fout: string);
       n := Pos('$VERSION', s);
     end;
 
+    n := Pos('$RELEASE', s);
+    while n > 0 do
+    begin
+      Delete(s, n, 8);
+      Insert(SKeymanVersion, s, n);
+      n := Pos('$RELEASE', s);
```

Comment:
The convention seems to be `$RELEASE` (10.0) is the short portion of $VERSION (e.g. 10.0.700).
Can that be noted somewhere so future maintainers know when to use $RELEASE vs $VERSION?

Reply:
This information is already in the mkver (aka KMVER) commandline help:
```
KMVer: Keyman project version information increment

Usage: kmver -i [-d] <template> <root>
 or    kmver -v [-u f.in f.out]* <template> [version.rc]
   -i:         Updates the template build version from src\version.txt
   -d:         Debug version
   -v:         Update the version.rc with the template information
   -m:         Update manifest.xml with the template information
   -u:         Update file f.in to f.out, replacing (multiple entries okay):
                          $VERSION     11.0.700.0
                          $VERSIONNUM  9,0,700,0
                          $VERSIONCVS  9-0-700-0
                          $RELEASE     11.0
                          $GUID#       GUID 0-9
                          $DATE        1 March 2018
   template:   The source template, usually \keyman\11.0\src\<dir>\version.txt
   root:       The source template, usually \keyman\11.0\src\version.txt
   version.rc: version.rc file to update
```